packages/server/package.json

```diff
@@ -25,10 +25,11 @@
     "prepublishOnly": "yarn run build"
   },
   "peerDependencies": {
-    "@loadable/component": "^5.0.1",
+	"@loadable/component": "^5.0.1",
+	"react-prerendered-component": "^1.2.4",
```

Comment:
I'm not sure at all how to define this dependency, as it's required in both client and server. It can be a dependency, but loadable/component already has it, so i put it as peer dependency to be aligned with it.